### PR TITLE
Updated identity docs

### PR DESCRIPTION
**Pull request by martinlissmyr**

### CODE REVIEW

**Review by arvida — APPROVED** [review]
From my point of view, it looks like most of the hopp-identity is covered here. 

I have added a few questions and notes. I guess we haven't really discussed the docs before, so I am a bit unsure about some of the things here and how they should be interpreted. 

Anyway, great keeping these things documented!

File: docs/identity/index.md [hunk 1]
```diff
@@ -6,129 +6,5 @@ has_children: true
...
-* `signup_completion_page`
-* `signin_page`
-* `password_reset_page`
-* `profile_page`
-* `profile_edit_page`
-* `profile_password_change_page`
-
-
 
+A system for handling product offers and subscribers for an organisation. The purpose of this system is to tie the different parts of the product experience together for the customer.
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/index.md:10]: Fo sho.

File: docs/identity/subscriptions.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+layout: default
+title:  "Subscriptions"
+nav_order: 100
+has_children: false
+parent: "Identity"
+---
+
+# Subscriptions
+
+A subscription represents a current or historic connection between a customer and a package (and hence products). A subscription will only last as long as the validity period of the connected package. After that a new subscription will be created if a rollover package is defined and rollovers aren't specifically prohibited. This means loyal customers will have several successive subscriptions to a package. This is a technicality and should be taken into consideration when presenting information to end users as they  most likely view this, from their perspective, as *one* ongoing subscription.
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/subscriptions.md:11]: ✅

File: docs/identity/users-customers-administrators.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+# Users, Customers & Administrators
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+## Customer
+
+A customer is an identity who potentially will, has had or currently have a relationship to a product. It will have:
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/users-customers-administrators.md:17]: ✅

File: docs/identity/users-customers-administrators.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* Two-Factor Authentication (optional)
+
+## Identity
+
+An identity may have:
+
+* A user
+* One or several subscriptions
+* Name & address
+* E-mai
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/users-customers-administrators.md:45]: 🙀

File: docs/identity/page-types.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title:  "Page Types"
+nav_order: 400
+parent: "Identity"
+---
+
+# Page Types
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/page-types.md:8]: It's supposed to be used in analytics mostly. There's a todo to exposed these to Google Analytics as a custom dimension.

File: docs/identity/user-features.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title:  "User Features"
+nav_order: 200
+parent: "Identity"
+---
+
+# User Features
```
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/user-features.md:8]: Yep, it's incomplete... I agree on the paywall thing, that's why it was linked elsewhere. But we can remove it completely.

File: docs/identity/page-types.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (arvida) [reply docs/identity/page-types.md:8]: Aha ok!

File: docs/identity/user-features.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (martinlissmyr) [reply docs/identity/user-features.md:8]: I won't add "new documentation" to this PR. The purpose here was mostly to update the existing docs and make sure they where correct. I will create a new PR for additions to user and admin features when this is merged.

**Pull request merged**